### PR TITLE
[KEYCLOAK-12873] Fix differences between keycloak-saml.xml (adapter) …

### PR DESCRIPTION
**Pull request by sguilhen**
…and the keycloak-saml subsystem

Link to Jira: https://issues.redhat.com/browse/KEYCLOAK-12873

### CODE REVIEW

File: adapters/saml/as7-eap6/subsystem/src/main/resources/org/keycloak/subsystem/saml/as7/LocalDescriptions.properties [hunk 1]
```diff
@@ -68,6 +69,7 @@ keycloak-saml.IDP.remove=Remove an identity provider
 keycloak-saml.IDP.signaturesRequired=Require signatures for SingleSignOnService and SingleLogoutService
 keycloak-saml.IDP.signatureAlgorithm=Signature algorithm
 keycloak-saml.IDP.signatureCanonicalizationMethod=Signature canonicalization method
+keycloak-saml.IDP.metadataUrl=he URL used to retrieve the IDP metadata from
```
Comment (hmlnarik) [inline adapters/saml/as7-eap6/subsystem/src/main/resources/org/keycloak/subsystem/saml/as7/LocalDescriptions.properties]:
```suggestion
keycloak-saml.IDP.metadataUrl=The URL used to retrieve the IDP metadata from
```

**Comment by hmlnarik** [comment]
Thanks @sguilhen ! At the first look looks good

**Review by mhajas — APPROVED** [review]
@sguilhen Thanks! LGTM

**Review by hmlnarik — APPROVED** [review]
Thanks @sguilhen and @mhajas ! Approving based on @mhajas 's review

**Pull request merged**